deps/npm/LICENSE

```diff
@@ -1,11 +1,29 @@
+The npm application
 Copyright (c) npm, Inc. and Contributors
-All rights reserved.
+Licensed on the terms of The Artistic License 2.0
```

Comment:
> You mean All rights reserved. needs to remain?

No... that would **not** be leveraging the Copyright... which means *node* couldn't distribute it period as a module without a written agreement on paper registered with a legal authority... that costs $$ to get it registered and pretty much defeats FOSS.